pylint/message/deleted_message_ids.py

```diff
@@ -0,0 +1,146 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+from functools import lru_cache
+from typing import NamedTuple
+
+
+class DeletedMessage(NamedTuple):
+    msgid: str
+    symbol: str
+    old_names: list[tuple[str, str]] = []
+
+
+_DELETED_MSGID_PREFIXES: list[int] = []
```

Comment:
If we underscore this file we can then not underscore anything else in this file 😄

Reply:
Clever 😄

Reply:
Done